src/SleepForDays/SleepForDays.workflow.cs

```diff
@@ -0,0 +1,33 @@
+namespace TemporalioSamples.SleepForDays;
+
+using Microsoft.Extensions.Logging;
+using Temporalio.Workflows;
+
+[Workflow]
+public class SleepForDaysWorkflow
+{
+    private bool isComplete;
+
+    [WorkflowRun]
+    public async Task RunAsync()
+    {
+        isComplete = false;
+        while (!isComplete)
+        {
+            await Workflow.ExecuteActivityAsync(
+                (SleepForDaysActivities act) => act.SendEmail("Sleeping for 30 days"),
+                new() { StartToCloseTimeout = TimeSpan.FromSeconds(10) });
+            await Task.WhenAny(
```

Comment:
```suggestion
            await Workflow.WhenAnyAsync(
```

Even though `Task.WhenAny(` technically works here in most cases, we encourage the safer wrapper

Reply:
Didn't realize there was a workflow wrapper, switched.

Reply:
Make sure it's `WhenAnyAsync`

Reply:
Updated